docker/wait-for

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+# Downloaded from https://raw.githubusercontent.com/eficode/wait-for/master/wait-for
```

Comment:
I think we should move wait-for to its own folder so that we can properly include its [MIT license](https://github.com/eficode/wait-for/blob/master/LICENSE)

Reply:
Just did it. I Created the folder, moved the script there, and downloaded the LICENSE from its github page.